maven/jdk17/Dockerfile.nanoserver

```diff
@@ -0,0 +1,21 @@
+# escape=`
+ARG JAVA_VERSION=11.0.17_8
```

Comment:
Shouldn't all these java & jdk versions be 17.x in this file?

Reply:
We need both:

- JDK17 from the inbound image, as the default JDK17 used by tthe builds (variable `JAVA_HOME` + binary `java` in the Path).
- JDK11 on a fixed location to allow execution of the agent process (that should be the same JDK as the controller).

Reply:
Oh right, sorry my mistake, nevermind my comment.